code/style.css

```diff
@@ -1,58 +1,264 @@
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 100;
 }
 
 .killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
+  width: 500px;
+  height: 550px;
   text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+  margin-right: 30px;
+  border-radius: 5px;
+  background-image: url("./assets/diamond-background-card.png");
+  background-size: 20px;
+  cursor: pointer;
+  box-shadow: 0 3px 5px 0 #555252;
+}
+
+.killer-deck p {
+  margin: 0px;
 }
 
 .weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
-  text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+  width: 500px;
+  height: 550px;
+  margin-right: 30px;
+  border-radius: 5px;
+  background-image: url("./assets/diamond-background-card.png");
+  background-size: 20px;
+  background-repeat: repeat;
+  cursor: pointer;
+  box-shadow: 0 3px 5px 0 #555252;
 }
 
-.room-deck {
-  width: 250px;
-  height: 400px;
-  background: midnightblue;
+.weapon-deck p {
+  margin: 0px;
+}
 
+.room-deck {
+  width: 500px;
+  height: 550px;
   text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+  border-radius: 5px;
+  background-image: url("./assets/diamond-background-card.png");
+  background-size: 20px;
+  cursor: pointer;
+  box-shadow: 0 3px 5px 0 #555252;
+}
+
+.room-deck p {
+  margin: 0px;
+}
+
+.card-front {
+  display: block;
+  justify-content: center;
+  align-items: center;
+  margin-top: 30px;
+}
+
+.card-front img {
+  display: block;
+  margin-top: 50px;
+  margin-left: 165px;
+  border-radius: 50%;
+  padding: 10px 5px;
+  border-radius: 50%;
 }
 
+.loader-image {
+  background: #cccccc;
+}
+
+.animated-loader {
+  animation-name: spinningglass;
+  animation-duration: 2s;
+  animation-timing-function: linear;
+}
+
+@keyframes spinningglass {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+.card-title {
+  background: #f2f2f2;
+  display: inline;
+  border-radius: 5px;
+  padding: 10px 20px;
+  color: #000000;
+  font-family: 'Prata', serif;
+  font-size: 20px;
+}
+
+.card-title h4 {
+  margin-top: 0px;
+  padding-top: 0px;
+}
+
+.card-details {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border: 4px solid #f2f2f2;
+  height: 90%;
+  margin: -15px 15px 15px 15px;
+  border-radius: 5px;
+}
+
+.card-details h4 {
+  border-bottom: 2px solid #000000;
+  padding-bottom: 3px;
+  color: #000000;
+}
+
+.images {
+  border: 4px solid #f2f2f2;
+  border-radius: 2px;
+  padding: 5px;
+}  
+
 button {
-  padding: 8px 16px;
-  background: crimson;
+  padding: 10px;
+  background: #333333;
+  color: #ffffff;
   border: none;
-  font-size: 20px;
-  border-radius: 8px;
+  font-size: 18px;
+  border-radius: 3px;
   display: block;
   margin: 0 auto;
+  font-family: 'Prata', serif;
+}
+
+button:hover {
+  background: #ffffff;
+  color: #000000;
+  cursor: pointer;
 }
+
+.reveal-mystery-container {
+  background: #cccccc;
```

Comment:
Final note; if you have a six-digit hex code which consists of identical characters, you can cut it in half. 

I.e. "#000000" is exactly the same as "#000", and "#cccccc" is exactly the same as "#ccc".

Not a major thing, but seeing as how it looks like you'll have a long and successful career of front-end development ahead of you – it can potentially save you MINUTES in the long run 🙂

Reply:
Good job Axel!

Reply:
Totally make sense. Thank you @EmOlsz and Axel! I just want to say that I think the quality of notes from Axel were really clear and thorough. So thank you!